pom.xml

```diff
@@ -474,12 +428,6 @@
         </exclusions>
       </dependency>
       <dependency>
-        <groupId>org.javassist</groupId>
```

Comment:
FYI, sonar for `fcrepo4` has been disabled until this ticket is completed:
https://jira.duraspace.org/browse/FCREPO-1670